src/libs/ReportUtils.js

```diff
@@ -1278,16 +1278,20 @@ function getDisplayNameForParticipant(accountID, shouldUseShortForm = false, sho
     if (!accountID) {
         return '';
     }
+
     const personalDetails = getPersonalDetailsForAccountID(accountID);
-    // this is to check if account is an invite/optimistically created one
+
+    // This is to check if account is an invite/optimistically created one
     // and prevent from falling back to 'Hidden', so a correct value is shown
-    // when searching for a new user
+    // when searching for a new user while offline
     if (lodashGet(personalDetails, 'isOptimisticPersonalDetail') === true) {
-        return personalDetails.login || '';
+        return LocalePhoneNumber.formatPhoneNumber(personalDetails.login || '');
     }
-    const longName = personalDetails.displayName;
+
+    const formattedLogin = LocalePhoneNumber.formatPhoneNumber(personalDetails.login || '');
```

Comment:
Move this before `if (lodashGet(personalDetails, 'isOptimisticPersonalDetail') === true) {` and reuse in return value

Reply:
Updated.